client/StatusTable/components/StatusTable.js

```diff
@@ -0,0 +1,66 @@
+import React, {Component} from 'react'
+import PropTypes from 'prop-types'
+import {connect} from 'react-redux'
+import * as actions from '../actions'
+import WeekRow from './WeekRow'
+
+class StatusTable extends Component {
+    componentDidMount() {
+        this.props.tableLoadInit();
```

Comment:
try useEffect there. And may there is better place to dispatch init action than from the table itself